src/components/features/Promotions/Promotions.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Promotions.module.scss';
+
+class Promotions extends React.Component {
+  render() {
+    const { promotions } = this.props;
+
+    return (
+      <div className={styles.root}>
+        <div className='container'>
+          <div className='row'>
+            <div className='col'>
+              <div className={styles.bigLeft}>
+                <img
+                  className={styles.image}
+                  alt={promotions[0].alt}
+                  src={promotions[0].image}
+                />
+                <div className={styles.overlay}></div>
+                <div className={styles.opacityText}>
+                  <p>{promotions[0].text}</p>
+                  <span>{promotions[0].text_bold}</span>
```

Comment:
Are you sure a <p> tag is a suitable choice for these headings? ;)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Paragraphs are considered to be long text, like multi-line content. For short texts use headings (if it's a heading semantically), span or no tag at all.